pages/profile/edit-profile.tsx

```diff
@@ -0,0 +1,115 @@
+import styles from '../../styles/Page.module.css'
+import profileStyles from '../../styles/Profile.module.css'
+import TopNav from '../../components/TopNav'
+import ProfilePicture from '../../components/profile/ProfilePicture'
+import Button from '../../components/Button'
+import React, { useEffect, useState } from 'react'
+
+// Credits to https://medium.com/web-dev-survey-from-kyoto/how-to-customize-the-file-upload-button-in-react-b3866a5973d8 for
+// the styling of the photo upload button
+const AlertMessage = ({ messageType, messageContent, visible }) => {
+  if (visible) {
+    return <div className={profileStyles[messageType]}>{messageContent}</div>
+  } else {
+    return null
+  }
+}
+
+// TO-DO: Alert message should display error if there is error in saving info (instead of always displaying saved successfully)
```

Comment:
nit: `TODO` is more standard than `TO-DO` in tooling.
```suggestion
//TODO: Alert message should display error if there is error in saving info (instead of always displaying saved successfully)
```